addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -219,6 +228,80 @@ odoo.define('website_slides.quiz', function (require) {
             });
         },
 
+        /**
+         * @private
+         */
+        _onClicksigninJoin: function (event) {
```

Comment:
This looks very (very) similar to `_onClickJoinSale`

You should try to factorize the common parts.